python/cudf/cudf/utils/ioutils.py

```diff
@@ -1352,6 +1353,10 @@ def get_filepath_or_buffer(
     open_file_options : dict, optional
         Optional dictionary of key-word arguments to pass to
         `_open_remote_files` (used for remote storage only).
+    is_raw_text_like_input : boolean, default False
+        If True, this determines the input `path_or_data` could be
+        a JSON string, hence will not check for it's existence in file-system.
+        If False, will check for it's existence and raise error.
```

Comment:
We aren't doing any JSON-specific checks here so we shouldn't declare that it's JSON only. Can users read CSV or other text formats from a string?
```suggestion
    allow_raw_text_input : boolean, default False
        If True, this indicates the input `path_or_data` could be a raw text
        input, hence will not check for its existence in the filesystem. If
        False, the input must be a path and an error will be raised if it does
        not exist.
```